modules/android/pages/working-with-vector-search.adoc

```diff
@@ -233,32 +232,133 @@ include::example$kotlin_snippets/app/src/main/kotlin/com/couchbase/codesnippets/
 NOTE: You can use less storage by using the `prediction()` function as the encoded vectors will only be stored in the index. 
 However, the index time will be longer as vector embedding generation is occurring at run time.
 
-== Vector SQL++ Functions
+== Create a Lazy Vector Index
 
-Couchbase Lite currently supports two SQL++ functions, `vector_match()` and `vector_distance()`.
+Lazy indexing is an alternate approach to using the standard predictive model with regular vector indexes which handle the indexing process automatically.
+You can use lazy indexing to use a ML model that is not available locally on the device and to create vector indexes without having vector embeddings in the documents.
```

Comment:
I could be wrong about this. 
```suggestion
You can use lazy indexing to use an ML model that is not available locally on the device and to create vector indexes without having vector embeddings in the documents.
```

Reply:
I get what you mean. I'll leave as is for now as I _think_ a machine learning model reads correctly compared to an ❓ 😖